lib/signs/utilities/headways.ex

```diff
@@ -21,9 +25,33 @@ defmodule Signs.Utilities.Headways do
     end
   end
 
-  defp fetch_headways(headway_group, sources, current_time) do
-    stop_ids = Enum.map(sources, & &1.stop_id)
+  # Process headways for Silver Line differently b/c of its different config shape
+  @spec headway_message_sl(
+          String.t(),
+          PaEss.destination() | nil,
+          [String.t()] | nil,
+          DateTime.t()
+        ) ::
+          Message.t() | nil
+  def headway_message_sl(_headway_group, _headway_destination, stop_ids, _current_time)
+      when stop_ids == nil or stop_ids == [],
+      do: nil
```

Comment:
nit: This may be overly cautious, since I don't think `stop_ids` should ever be `nil`, and the downstream code looks like it will accept and empty array and return `false`.